feat(jobs): permissions precisions (#5657)
scaleway/docs-content: pages/serverless-jobs/how-to/run-job.mdx

    - A start date and an end date
    - The duration of the job execution

## Permissions and product dependencies

Depending on the settings used, Serverless Jobs can have dependencies on other products, such as:

> - **Secret Manager**: to securly inject data into jobs

Reviewer: ```suggestion
- **Secret Manager**: to securely inject data into jobs
```

- **Container Registry**: to store images of jobs

Executing a Serverless Job can fail if some permissions are not definied properly in an [IAM policy](/iam/how-to/create-policy/), for example:

- If the job definition uses an image from **Container Registry**, add the `ContainerRegistryReadOnly` permission.

> - If the job definition consumes data from **Secret Manager**; add the `SecretManagerSecretAccess` permission.

Reviewer: ```suggestion
- If the job definition consumes data from **Secret Manager**, add the `SecretManagerSecretAccess` permission.
```

    - A start date and an end date
    - The duration of the job execution

## Permissions and product dependencies

> Depending some settings Serverless Jobs can rely on different products like:

Reviewer 2: ```suggestion
Depending on various settings, Serverless Jobs can have dependencies on other products, for example:
```
